examples/_empty/src/model.py

```diff
@@ -0,0 +1,12 @@
+import torch
+from catalyst.contrib import registry
+
+
+@registry.Model
+class Model(torch.nn.Module):
```

Comment:
it would be nice if this example would work just out of the box
You already have datasets, so I suggest you to add CIFAR10 model here

Reply:
btw, even better idea – to add `get_from_params` method to have an example of model construction from config